src/pages/Account/Create.tsx

```diff
@@ -1,84 +1,61 @@
 import React from 'react';
 import Helmet from 'react-helmet';
-import ManageAccountContainer, { ManageAccountModes } from '../../features/Account/ManageAccountForm';
+import ManageAccountForm, {
+  ManageAccountModes,
+} from '../../features/Account/ManageAccountForm';
 
 import * as CONSTANTS from '../../config/constants';
-import {
-  BackgroundImage,
-  H1,
-  MaxWidthBox,
-} from '../../shared/Elements';
-import MediaQuery from 'react-responsive';
+import { H1 } from '../../shared/Elements';
 
-import Coders from '../../assets/images/coders.svg';
+import GirlAtHome from '../../assets/images/girl-at-home.svg';
 
 const CreateAccountPage: React.FC = () => (
   <>
     <Helmet>
-      <title>
-        Create Account | {CONSTANTS.HACKATHON_NAME}
-      </title>
+      <title>Create Account | {CONSTANTS.HACKATHON_NAME}</title>
     </Helmet>
 
-    <MediaQuery maxWidth="991px">
-      <MaxWidthBox m={'auto'} paddingLeft={'50px'} paddingRight={'50px'} maxWidth={'500px'}>
-        <H1
-          fontSize={'30px'}
-          textAlign={'left'}
-          marginTop={'0px'}
-          marginBottom={'20px'}
-          marginLeft={'0px'}
-          paddingBottom={'20px'}
-          paddingTop={'70px'}
-        >
-          Your Account
-        </H1>
-        <ManageAccountContainer mode={ManageAccountModes.CREATE} />
-      </MaxWidthBox>
-    </MediaQuery>
+    <div className="SignUp--form-container">
+      <div className="SignUp--art-wrapper">
+        <img src={GirlAtHome} className="SignUp--art" alt="Background" />
+      </div>
 
-    <MediaQuery minWidth="992px">
-      <MaxWidthBox paddingLeft={'100px'} paddingRight={'50px'} maxWidth={'500px'}>
-        <H1
-          fontSize={'30px'}
-          textAlign={'left'}
-          marginTop={'0px'}
-          marginBottom={'20px'}
-          marginLeft={'0px'}
-          paddingBottom={'20px'}
-          paddingTop={'70px'}
-        >
-          Your Account
-        </H1>
-        <ManageAccountContainer mode={ManageAccountModes.CREATE} />
-      </MaxWidthBox>
-    </MediaQuery>
+      <form className="SignUp--form">
```

Comment:
Why are you wrapping the `Form` div in a `form` div?

Reply:
To apply CSS properties to this form (`box-sizing`, `flex`, `margin-right`, `margin-left`) - could edit `Form.tsx` and make it accept that as props instead but I'd like to move away from inline css and having components that just replicate standard HTML without adding features. Like having `<H1 fontSize={'30px'} marginTop={'100px'} marginLeft={'0px'}>` isn't the best imo. Think that if we could have just one locally scoped style block at the bottom of every component that needed it, would remove the entanglement of structure and style, making stuff like media queries much cleaner.

Reply:
What about making the outer component a <div> and not a <form>? Therefore no nested <form> object.

Reply:
Oh yikes, missed the nested `<form>`s